foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/PythonLintingMigration.java

```diff
@@ -0,0 +1,137 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import com.boozallen.aissemble.upgrade.util.pom.LocationAwareMavenReader;
+import com.boozallen.aissemble.upgrade.util.pom.PomModifications;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.Build;
+import org.apache.maven.model.InputLocation;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.Plugin;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+import java.io.File;
+import java.util.List;
+
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.getLocationAnnotatedModel;
+import static com.boozallen.aissemble.upgrade.util.pom.PomHelper.writeModifications;
+
+/**
+ * This migration disables failing on Python linting for existing projects.
+ */
+public class PythonLintingMigration extends AbstractAissembleMigration {
+    private Plugin getHabushuPlugin(Model model) {
+        List<Plugin> plugins = model.getBuild().getPluginManagement().getPlugins();
+        for (Plugin plugin : plugins) {
+            if ("habushu-maven-plugin".equalsIgnoreCase(plugin.getArtifactId())) {
+                return plugin;
+            }
+        }
+        return null;
+    }
+
+    private Xpp3Dom getConfiguration(Plugin plugin) {
+        Object lintingConfiguration = plugin.getConfiguration();
+        return (Xpp3Dom) lintingConfiguration;
+    }
+
+    private boolean hasLinting(Xpp3Dom configuration) {
+        if (configuration.getChildren("sourceFailOnLintErrors").length > 0 || configuration.getChildren("testFailOnLintErrors").length > 0) {
+            return true;
+        }
+        return false;
+    }
+
+    private boolean isRootProjectPom(Model model) {
+        return "com.boozallen.aissemble".equalsIgnoreCase(model.getParent().getGroupId());
```

Comment:
C: We actually need to check the artifactId is `build-parent` as well here.

Reply:
Great catch, thanks Emily!